test/connect.config

```diff
@@ -2,11 +2,10 @@
 %% ex: ft=erlang
 
 #{
-tns => <<"(DESCRIPTION=(ADDRESS_LIST=(ADDRESS="
```

Comment:
Are changes to this config on purpose or added by mistake ?

Reply:
On purpose!